database/migrations.php

```diff
@@ -35,7 +35,14 @@ function migate(mysqli $db)
     CREATE TABLE Items
     (
     id INTEGER AUTO_INCREMENT PRIMARY KEY,
-    username VARCHAR(40) NOT NULL
+    itemName VARCHAR(40) NOT NULL,
+    description VARCHAR(100) NOT NULL,
+    categoryId INT NOT NULL,
+    startPrice DECIMAL(7,2) NOT NULL,
+    reservePrice DECIMAL(7,2) CHECK (reservePrice > startPrice),
```

Comment:
We should probably make this a >= check, as a seller I could want to start an auction at 20 and be willing to sell at that price.

Reply:
I think the way I was thinking about it is that as reserve price is optional, you would only set it at higher than the starting price? But it might depend how we want to store it in the database - we could just leave it null if there's no reserve price, or have it default to the start price...?